src/main/java/org/sagebionetworks/web/client/presenter/EmailInvitationPresenter.java

```diff
@@ -0,0 +1,169 @@
+package org.sagebionetworks.web.client.presenter;
+
+import static com.google.common.util.concurrent.Futures.getDone;
+import static com.google.common.util.concurrent.Futures.whenAllComplete;
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
+import static org.sagebionetworks.web.client.DisplayUtils.getDisplayName;
+
+import javax.inject.Inject;
+
+import org.sagebionetworks.repo.model.MembershipInvtnSignedToken;
+import org.sagebionetworks.repo.model.MembershipInvtnSubmission;
+import org.sagebionetworks.repo.model.Team;
+import org.sagebionetworks.repo.model.UserProfile;
+import org.sagebionetworks.web.client.GlobalApplicationState;
+import org.sagebionetworks.web.client.PlaceChanger;
+import org.sagebionetworks.web.client.SynapseFutureClient;
+import org.sagebionetworks.web.client.SynapseJavascriptClient;
+import org.sagebionetworks.web.client.place.EmailInvitation;
+import org.sagebionetworks.web.client.place.LoginPlace;
+import org.sagebionetworks.web.client.place.Profile;
+import org.sagebionetworks.web.client.place.Synapse;
+import org.sagebionetworks.web.client.security.AuthenticationController;
+import org.sagebionetworks.web.client.view.EmailInvitationView;
+import org.sagebionetworks.web.client.view.users.RegisterWidget;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.shared.NotificationTokenType;
+
+import com.google.common.util.concurrent.FluentFuture;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.gwt.activity.shared.AbstractActivity;
+import com.google.gwt.event.shared.EventBus;
+import com.google.gwt.user.client.ui.AcceptsOneWidget;
+
+public class EmailInvitationPresenter extends AbstractActivity implements EmailInvitationView.Presenter, Presenter<EmailInvitation> {
+	private String encodedMISignedToken;
+	private EmailInvitationView view;
+	private RegisterWidget registerWidget;
+	private SynapseJavascriptClient jsClient;
+	private SynapseFutureClient futureClient;
+	private SynapseAlert synapseAlert;
+	private AuthenticationController authController;
+	private PlaceChanger placeChanger;
+
+	@Inject
+	public EmailInvitationPresenter(EmailInvitationView view,
+									RegisterWidget registerWidget,
```

Comment:
all this cleanup, and we get 9 tabs indentation here? :)

Reply:
This looks aligned when viewed with 4 space tabs. 
![2017-10-26-11 35 55](https://user-images.githubusercontent.com/7145349/32070682-fb2251ec-ba41-11e7-8691-a0169b22fd36.png)
Should I avoid this kind of indentation?

Reply:
I'd recommend having the first parameter on a newline, and just indent all of these dependencies with a single tab.  But it's trivial, so feel free to ignore :)